add StreamGroupInfo EntriesRead and Lag

tvdias opened the pull request:
Fixes #2467

mgravell commented on src/StackExchange.Redis/APITypes/StreamGroupInfo.cs:
```diff
@@ -33,4 +35,14 @@ internal StreamGroupInfo(string name, int consumerCount, int pendingMessageCount
+    /// <summary>
+    /// Total number of entries the group had read.
+    /// </summary>
+    public int? EntriesRead { get; }
```
Should this be int64?

tvdias replied:
Updated. Does it make sense to update `lag` for the same reason? Although that should unusual to have a consumer group being created and reading messages from the beggining on very large streams.

tvdias replied:
Updated. Does it make sense to update `lag` for the same reason? Although that should be unusual to have a consumer group being created and reading messages from the beginning on very large streams.

mgravell replied:
it sounds *unusual*, but if it is theoretically possible, we should probably make the `Lag` 64-bit too, I guess.

tvdias replied:
Fixed on c57fbfbd129a9b4aca88b81e4d44a6b4e329739d and c57fbfbd129a9b4aca88b81e4d44a6b4e329739d.

mgravell commented:
Suggest also:

- release notes update
- test if possible (if server version is blockage, fine)

tvdias commented:
> * release notes update
Fixed on 287bacd6c4380a63c75a7b1a0f268e92933b6fde

> * test if possible (if server version is blockage, fine)
Fixed on f0f0a1d1c460fb3152f97d71794996e2f6391e4c

tvdias commented:
Hello @mgravell  , can you please take a new look?

And please use the `squash it on merge` option, as I had created some dummy commits to trigger the builds (idk if I could manually do it).

NickCraver reviewed (approved):
Current is looking great - thank you!
